lib/lotus/model/migrator/connection.rb

```diff
@@ -0,0 +1,87 @@
+module Lotus
+  module Model
+    module Migrator
+      # Sequel connection wrapper
+      #
+      # Normalize external adapters interfaces
+      #
+      # @since x.x.x
+      # @api private
+      class Connection
```

Comment:
Although it's a private api It would be cool if you improve comments for all methods, what do you think @joneslee85